string objectName
int32 delta
---
int32 result
